.github/workflows/release.yml

```
@@ -121,6 +121,11 @@ jobs:
           echo "version=$VERSION" >> $GITHUB_OUTPUT
           if [ "$(git tag --list "$VERSION")" ]; then
             echo "released=true" >> $GITHUB_OUTPUT
+
+            # Set the `release_notes` output
+            echo 'release_notes<<EOF' >> $GITHUB_OUTPUT
+            echo "$(node script/get-release-notes.js)" >> $GITHUB_OUTPUT
+            echo 'EOF' >> $GITHUB_OUTPUT
           else
             echo "released=false" >> $GITHUB_OUTPUT
```

Looks like this is the wrong way around:

```suggestion
          else
            echo "released=false" >> $GITHUB_OUTPUT
            
            # Set the `release_notes` output
            echo 'release_notes<<EOF' >> $GITHUB_OUTPUT
            echo "$(node script/get-release-notes.js)" >> $GITHUB_OUTPUT
            echo 'EOF' >> $GITHUB_OUTPUT
```